web/__tests__/jsdom/layout.test.tsx

```
@@ -42,19 +42,23 @@ describe("DashboardLayout", () => {
     (useToast as jest.Mock).mockReturnValue({ toast: mockToast });
   });
 
-  it("renders the home link", () => {
-    render(<DashboardLayout children={<div>Content</div>} />);
-    expect(screen.getByText("EpiCourier")).toBeInTheDocument();
+  it("renders the sidebar", () => {
+    const { container } = render(<DashboardLayout children={<div>Content</div>} />);
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)